napari/_qt/layer_controls/_tests/test_qt_labels_layer.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+from qtpy.QtCore import Qt
+
+from napari._qt.layer_controls.qt_labels_controls import QtLabelsControls
+from napari.layers import Labels
+
+np.random.seed(0)
+_LABELS = np.random.randint(20, size=(10, 15))
+
+
+def test_changing_layer_color_mode_updates_combo_box(qtbot):
+    """Updating layer color mode changes the combo box selection"""
+    layer = Labels(_LABELS)
+    qtctrl = QtLabelsControls(layer)
+    qtbot.addWidget(qtctrl)
+
+    original_color_mode = layer.color_mode
+    assert original_color_mode == qtctrl.colorModeComboBox.currentText()
+
+    layer.color_mode = 'selected'
+    assert layer.color_mode == qtctrl.colorModeComboBox.currentText()
+
+
+def test_changing_combo_box_updates_layer_color_mode(qtbot):
+    """Selecting a new color mode via combo box updates layer's color mode"""
+    layer = Labels(_LABELS)
+    qtctrl = QtLabelsControls(layer)
+    qtbot.addWidget(qtctrl)
+    color_mode_comboBox = qtctrl.colorModeComboBox
+
+    index = color_mode_comboBox.findText('selected', Qt.MatchFixedString)
```

Comment:
This fails because we connect the `color_mode_comboBox` to `activated` rather than `current_index_changed`. `activated` only fires through user interaction, not programmatic changes. I tried to simulate a button click instead, similar to this image layer test [here](https://github.com/napari/napari/blob/62a046de2e5009f0c9137e3fd3999fbf902928ca/napari/_qt/layer_controls/_tests/test_qt_image_base_layer_.py#L42), but I cannot for the life of me figure out how to select the item for clicking, as it's not a button in the eyes of qt.

@sofroniewn perhaps you know how to simulate the click on a `QComboBox` item?

Reply:
On discussion with @jni have removed this test and only kept the test checking api changes reflect in gui.